lib/fixer/sqs_client.rb

```diff
@@ -0,0 +1,54 @@
+# -*- encoding: utf-8 -*-
+
+require 'aws-sdk-core'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.